code/obj/machinery.dm

```diff
@@ -23,7 +23,12 @@
 	var/tmp/processing_tier = PROCESSING_EIGHTH
 	var/tmp/current_processing_tier
 	var/tmp/machine_registry_idx // List index for misc. machines registry, used in loops where machines of a specific type are needed
+	var/const/base_tick_spacing = 10 // Machines proc every 1*(2^tier-1) seconds. Or something like that.
+	var/const/cap_base_tick_spacing = 300 //
```

Comment:
i wouldnt set these to const if i were you. being able to adjust these in game is good for debugging. and const vars dont really save on perf at all.
```suggestion
	var/base_tick_spacing = 10 // Machines proc every 1*(2^tier-1) seconds. Or something like that.
	var/cap_base_tick_spacing = 300 //
```

Reply:
also you dont say anything with this comment

Reply:
I mean, const variables /are/ faster